core/src/main/java/com/github/kaktushose/jda/commands/dispatching/middleware/Middleware.java

```diff
@@ -1,17 +1,17 @@
 package com.github.kaktushose.jda.commands.dispatching.middleware;
 
-import com.github.kaktushose.jda.commands.annotations.Implementation;
+
 import com.github.kaktushose.jda.commands.dispatching.context.InvocationContext;
 import com.github.kaktushose.jda.commands.dispatching.middleware.internal.Middlewares;
+import com.github.kaktushose.jda.commands.extension.Implementation;
 import net.dv8tion.jda.api.utils.messages.MessageCreateData;
 
 import java.util.function.Consumer;
 
 /// Middlewares run just before an interaction event gets dispatched. They are used to perform additional checks or add
-/// more info the [InvocationContext]. Either register them at the [Middlewares] or use the [Implementation]
-/// annotation. Middlewares can have different [Priorities][Priority] dictating their priority on execution.
+/// more info the [com.github.kaktushose.jda.commands.dispatching.context.InvocationContext]. Either register them at the [Middlewares] or use the [Implementation]
+/// annotation. Middlewares can have different [Priorities][com.github.kaktushose.jda.commands.dispatching.middleware.Priority] dictating their priority on execution.
```

Comment:
please replace with imports and also we shouldn't link `com.github.kaktushose.jda.commands.dispatching.middleware.internal.Middlewares`, which is internal, but the concrete builder method